portal/client-services/org.wso2.is.portal.user.client.api/src/main/java/org/wso2/is/portal/user/client/api/IdentityStoreClientServiceImpl.java

```diff
@@ -57,6 +63,12 @@
 public class IdentityStoreClientServiceImpl implements IdentityStoreClientService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(IdentityStoreClientServiceImpl.class);
+    private static final String USERNAME_CLAIM = "http://wso2.org/claims/username";
+    private static final String GROUPNAME_CLAIM = "http://wso2.org/claims/groupname";
```

Comment:
I'm just keeing the one @Pushpalanka is using in group creation since that is the parameter I need to capture here. Will change if that changes